components/edit_post_modal/edit_post_modal.tsx

```diff
@@ -92,30 +151,33 @@ class EditPostModal extends React.PureComponent {
         document.removeEventListener('paste', this.handlePaste);
     }
 
-    setShowPreview = (newPreviewValue) => {
+    setShowPreview = (newPreviewValue: any) => {
```

Comment:
Same here, any suggestions on how to type this?

Reply:
Looks like `newPreviewValue` should be a `boolean`

Reply:
haha oops, done!